code/game/gamemodes/objective.dm

```diff
@@ -517,20 +517,21 @@ var/list/potential_theft_objectives = subtypesof(/datum/theft_objective) \
 	if(owner.kills.len>5) return 0
 	return 1
 
-//Vox heist objectives.
+//RAIDER heist objectives.
 
 /datum/objective/heist
+
 /datum/objective/heist/proc/choose_target()
 	return
 
 /datum/objective/heist/kidnap
 /datum/objective/heist/kidnap/choose_target()
-	var/list/roles = list("Chief Engineer","Research Director","Roboticist","Chemist","Station Engineer")
+	var/list/roles = list("Head of Security", "Head of Personnel", "Captain", "Nanotrasen Representative")

```

Comment:
Any reason for changing this from technicians to command staff? To increase the challenge of the objective, or...?